### ISSUE TITLE
[web] Changing from "Reports" to "Statistics" wipes already set filters

### ISSUE THREAD

**Issue by whisperity** [Ericsson/codechecker#3828]
> Originally reported in #3286 **on April 26, 2021** in a slightly different way.

**Describe the bug**
The filter page for "Reports" and "Statistics" offer at least a common intersection of options. Setting some filters on one, however, gets lost when changing to the other view. (The use case was that I set some filters up on the "Reports" page but then wanted to get a statistics of **those** specific reports, and I had to redo the filters.)

Moving to the *"statistics"* page is hard-coded to lead someone to:

```
<PRODUCT_URL>/statistics/overview?is-unique=on&detection-status=New&detection-status=Reopened&detection-status=Unresolved
```

**The configured filters should be stored in the URL or somewhere in the GUI in a way that changing viewports between the same product retains these filters.**

**CodeChecker versions affected**
 - 6.21.0
 - 6.22.2

**Additional context**

According to @dkrupp in #3286, release 6.15.2 used to behave in a way that going **from** *Statistics* **to** *Reports* kept the filters, but not the other way around. In this case, the behaviour looks like both a bug AND a regression.

**Comment by vodorok** [comment]
These components are not part of the keepalive component list. Should be an easy fix, after #3840
